sqlalchemy_seed/__init__.py

```diff
@@ -90,6 +92,22 @@ def _create_model_instance(fixture):
     return instances
 
 
+def _create_table_object_data(fixture, session):
+    """Create a Table object entry.
+
+    :param fixture: Fixtures
+    """
+    for data in fixture:
+        if 'table' in data:
+            module_name, class_name = data['table'].rsplit('.', 1)
+            importlib.import_module(module_name)
+            metadata = MetaData()
+            metadata.reflect(bind=session.get_bind())
+            table = metadata.tables[class_name]
+            insert = table.insert()
+            session.execute(insert.values(**data['fields']))
```

Comment:
I'm not familier with `Table` so, if I say something wrong with this review, please point out 😉 
What happen if insert failed. Does DB rollback?

What about do something like followings?
Is this satisfy your requirements?
```python
def _create_table_object_data(fixture, session):
    """Create a Table object entry.

    :param fixture: Fixtures
    :param session: Session
    """
    table_values = []
    for data in fixture:
        if 'table' in data:
            module_name, class_name = data['table'].rsplit('.', 1)
            importlib.import_module(module_name)
            metadata = MetaData()
            metadata.reflect(bind=session.get_bind())
            table = metadata.tables[class_name]
            insert = table.insert()
            insert.values(**data['fields'])
            table_values.append(insert)

    return table_values

def load_fixtures(session, fixtures):
    """Load fixture.

    :param base: `sqlalchemy.ext.declarative`
    :param fixtures: Fixture files
    """
    instances = []
    for fixture in fixtures:
        _instances = _create_model_instance(fixture)
        _tables = _create_table_object_data(fixture, session)
        for instance in _instances:
            instances.append(instance)

    try:
        for instance in instances:
            session.merge(instance)

        for insert in _tables:
            session.execute(insert)
        session.flush()
        session.commit()
    except Exception:
        session.rollback()
        raise
```

Reply:
nn- I think I wrote something strange...